libs/dialog/src/lib/dialog.component.scss

```diff
@@ -97,6 +99,6 @@
   }
 }
 
-body.ngneat-dialog-hidden {
+body.ngneat-dialog-hidden:not(.overflow) {
   overflow: hidden;
```

Comment:
We can use a CSS variable instead of a class:

 overflow: var(--dialog-overflow, hidden);